lib/core/include/irods/rodsErrorTable.h

```diff
@@ -161,9 +161,6 @@ NEW_ERROR(SYS_DIR_IN_VAULT_NOT_EMPTY,                  -82000)
 NEW_ERROR(SYS_OPR_FLAG_NOT_SUPPORT,                    -83000)
 NEW_ERROR(SYS_TAR_APPEND_ERR,                          -84000)
 NEW_ERROR(SYS_INVALID_PROTOCOL_TYPE,                   -85000)
-NEW_ERROR(SYS_UDP_CONNECT_ERR,                         -86000)
-NEW_ERROR(SYS_UDP_TRANSFER_ERR,                        -89000)
-NEW_ERROR(SYS_UDP_NO_SUPPORT_ERR,                      -90000)
```

Comment:
Unfortunately, I don't think we can get rid of these... Maybe we can leave a comment saying these aren't used anymore? We have this `END_OF_LIFE_` prefix that was used with the old xmsg server error codes, but not sure if we want to use that.

Reply:
I'll bring them back with just a comment. If we wish to use `END_OF_LIFE_` we can always change it.

Reply:
Brought the codes back. They are now blocked and stated to be EOL.